File: Modul-4/Soal 5/PRAK405-2410817120019-NadyaRamadhani.c
#include <stdio.h>
int main(){
    int n, kelipatan;
    int total = 0;

    printf("");
    scanf("%d %d", &n, &kelipatan);

    for(int i=1; i<=n; i++){
        int z = 0;
        for(int j=i; j>=1; j--){
            z += j*kelipatan;
            printf("(%d * %d)", j, kelipatan);
            if(j>1){
                printf(" + ");
            }
        }
        printf(" = %d\n", z);
        total += z;
    }
    printf("%d", total);

    return 0;
}

File: Modul-4/Soal 5/PRAK405-2410817120019-NadyaRamadhani.py
n, kelipatan = map(int, input("").split( ))
total = 0

for i in range (1, n+1):
    z = 0
    for j in range (i, 0, -1):
        z += j*kelipatan
        print(f"({j} * {kelipatan})", end=' ')
        if j>1:
            print(" + ", end=' ')

    print(f" = {z}")
    total += z

print(total)